### DISCUSSION TITLE
Fancybox version 4 data-height and data-width issue.

### DISCUSSION THREAD

**Discussion by vatsalribadiya** [fancyapps/ui#120] [General]
I have upgraded fancybox v3 to v4. Now i am facing issue in data-height and data-width. When i am trying to set height and width for fancybox iframe through passing data-height and data-width attributes , it is not getting applied. can you please help me out regarding this issue.

Thank you!

**Comment by fancyapps** [comment]
Hi,

Sorry for the late reply, support for data-width and data-height attributes was added into previous releases, see this demo - https://codesandbox.io/s/jolly-shamir-p5kdu?file=/index.html

**Comment by ThaisCursi** [comment]
Hello.
By setting the measurements I am limited in terms of screen responsiveness. How could I go about setting the measurements for that specific popup and making it responsive?

**Reply by fancyapps** [reply]
@ThaisCursi Simply disable auto-sizing and then use CSS, see second example here - https://codesandbox.io/s/jolly-shamir-p5kdu?file=/index.html

HTML 
```html
<a
  href="./src/iframe.html"
  data-fancybox
  data-type="iframe"
  data-id="iframeB"
  data-auto-size="false"
>
  Example
</a>
```
CSS
```css
#iframeB {
  max-width: 480px;
  max-height: 320px;
}
```

**Comment by ThaisCursi** [comment]
Hi Janis Skarnelis.

Got it, thanks for the feedback.
